### DISCUSSION TITLE
Drastic menu button, possible bug?

### DISCUSSION THREAD

**Discussion by SirPrimalform** [OnionUI/Onion#1576] [Support]
When Drastic, what if anything, is the Menu key supposed to do when pressed on its own? In the online manual there isn't a function listed for the menu key in Drastic, so I'd expect it to either do nothing or take me back to the launcher like for other emulators. Instead it seems to do a quick-load, but only the first time. Subsequent presses don't do anything until the emulator is restarted.
Seems like a bug, but I wanted to check I'm not being an idiot before reporting it.

**Comment by theUnBurn** [comment]
I actually been dealing with this for a while, and it's most likely because you set "Swap L1-L2" enabled in Drastic Settings. If you disable this, it should work as intended. 

I made a fix here for it, and so hopefully will be fixed in release soon: https://github.com/OnionUI/Onion/pull/1582
